src/components/chip/react/__snapshots__/Chip.test.tsx.snap

```diff
@@ -3,7 +3,7 @@
 exports[`<Chip /> Snapshot should render correctly Chip component 1`] = `
 <a
   LabelComponent="Hello World!"
-  className="lumx-chip lumx-chip--size-m lumx-chip--theme-light lumx-chip--is-unselected"
+  className="lumx-chip lumx-chip--color-dark lumx-chip--size-m lumx-chip--is-unselected"
```

Comment:
So no theme by default is actually what we want?

Reply:
Yep :3